jsonarchive.exe.d.json

```diff
@@ -19,9 +19,15 @@
     },
     "chunkSize": {
       "displayName": "Chunk size",
-      "helpText": "The maximum size, in bytes, for each file in the set. The default is 100 MB.",
+      "helpText": "The maximum size, in bytes, for each file in the set. The default is 104857600 (100 MB).",
       "isOptional": true,
       "inputType": "integer" 
+    },
+    "readableFilenames": {
```

Comment:
Perhaps this property could come directly after the `fileSet` property, so that in the UI, the checkbox is right next to the other option that controls file name formatting?